compiler/rustc_feature/src/unstable.rs

```diff
@@ -221,6 +221,8 @@ declare_features! (
     (internal, profiler_runtime, "1.18.0", None),
     /// Allows using `rustc_*` attributes (RFC 572).
     (internal, rustc_attrs, "1.0.0", None),
+    /// Keep UB checks for the standard library.
+    (internal, rustc_ub_check, "CURRENT_RUSTC_VERSION", None),
```

Comment:
Shouldn't this just `rustc_attrs` rather than introducing a new feature gate?

Reply:
Thanks, I know how to write it now.